vevestaX/vevesta.py

```diff
@@ -789,46 +802,72 @@ def __EDAForPandas(self, fileName):
         [x.tick_params(axis='x', labelrotation=90) for x in fig.ravel()]
         plt.plot()
         plt.suptitle('Feature Histogram',fontsize=20)
-        plt.savefig(os.path.join(directoryToDumpData, FeatureHistogramImageFile), bbox_inches='tight', dpi=100)
+        plt.savefig(join(directoryToDumpData, FeatureHistogramImageFile), bbox_inches='tight', dpi=100)
         plt.close()
+        
+        #Probability Density Function
+        numericDataframe = self.__data.select_dtypes(include='number')  
+        if self.__ycolumn!=None:
+            k=1
+            fig = plt.figure(figsize=(20,15))
+            for i in numericDataframe:
+                if i!=self.__ycolumn:
+                    ax = fig.add_subplot(4,(len(numericDataframe.columns)//4)+1, k)
+                    frequency=self.__data[self.__ycolumn].value_counts()
+                    frequency=dict(frequency)
+                    frequency=list(frequency.keys())[0:10]
```

Comment:
add a check if the value isnt int or string but float then u shouldnt plot it.

Reply:
i think you can use this as regression filter

Reply:
yes maam i have added the filter